profiles/kentik_snmp/cisco/cisco-all-devices.yml

```diff
@@ -913,6 +914,20 @@ metrics:
         column:
           name: entPhysicalModelName
           OID: 1.3.6.1.2.1.47.1.1.1.1.13
+  
+  # This table contains the additional information for the main PSE group in pethMainPseTable
+
+  - MIB: CISCO-POWER-ETHERNET-EXT-MIB
+    table:
+      name: cpeExtMainPseTable
+      OID: 1.3.6.1.4.1.9.9.402.1.3
+    symbols:  
+      - name: cpeExtMainPseUsedPower #  This object indicates used power expressed in miliwatts 
+        OID: 1.3.6.1.4.1.9.9.402.1.3.1.4
+        conversion: milliwatts to watts
+      - name: cpeExtMainPseRemainingPower #  This object indicates remaining power expressed in miliwatts, this parameter is calculated as pethMainPsePower minus cpeExtMainPseUsedPower 
+        OID: 1.3.6.1.4.1.9.9.402.1.3.1.5
+        conversion: milliwatts to watts
```

Comment:
this is not a valid conversion; redact this line and you can do the conversion on the raw data once it's shipped to the target sink

Reply:
@thezackm I've removed the conversion lines. Are any other changes required in this file? Also, wondering if it's possible to get a percentage of power used or available using these OIDs?